module/category-tree/tests/Domain/ValueObject/NodeTest.php

```diff
@@ -51,12 +51,17 @@ public function testSettingParent(): void
 
     /**
      */
-    public function testAddChildren():void
+    public function testAddChildren(): void
     {
+        $categoryId2 = new CategoryId('cde02652-70ce-484e-bc9d-3bf61391522d');
```

Comment:
Its betet use MockedObject

Reply:
When I use MockObject, don't know why the assertion on 'hasChildren' method fails.